### PR TITLE
Added backup guide

### PR DESCRIPTION
**Pull request by JacobHin2**
Have I done it wrong because I'm seeing a lot of files. Should I be committing without building the docs?

Anyway I added a guide on backing up your wallet.

### CODE REVIEW

File: source/conf.py [hunk 1]
```diff
@@ -42,7 +42,7 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
 
-html_title = 'Qogecoin docs'
```
Comment (neqochan) [inline source/conf.py:45]: I think all the files are generated because this makes global changes. Could you try to send this line as a separate commit?

**Review by neqochan — CHANGES_REQUESTED** [review]

File: source/conf.py (hunk 1 above)
Comment (neqochan) [reply source/conf.py:45]: Actually never mind i will fix this line later. Just send it without this line then changes should be smaller.

Comment (JacobHin2) [reply source/conf.py:45]: Alright.

**Comment by JacobHin2** [comment]
I'll just edit in GitHub maybe that will cause less changes.
